extract_wheels/lib/wheel.py

```diff
@@ -147,7 +147,13 @@ def extract_wheel(wheel_file: str, extras: List[str]) -> str:
                 bazel.sanitise_name(whl.name),
                 [
                     '"//%s"' % bazel.sanitise_name(d)
-                    for d in sorted(whl.dependencies(extras_requested=extras))
+                    for d in sorted(
+                        whl.dependencies(
```

Comment:
`build_file.write(...` begins a block of code that has some deeply nested function calls. Think this would be more readable after pulling the parts out. 

At the moment the code within the `with` block is 

```
 f(g(h(whl.name), [j(d) for d in k(m(...))]))
```

Better would be more like: 

```python
sanitised_dependences = [j(d) for d in k(m(...))]
contents = bazel.generate_build_file_contents(
    name=bazel.sanitise_name(whl.name),
    deps=sanitised_dependencies,
)
build_file.write(contents)
```